Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -211,11 +266,11 @@ ParserQuery: |
         , DnsQueryType between (261 .. 32767), 'Unassigned'
         , 'Unassigned'),
       EventResult=iff (EventResult == "Based on RCODE", iff(DnsResponseCode == 0, "Success", "Failure"),EventResult)
-   | extend
+  | extend
```

Comment:
do you see 'SourceName_s' in your logs? Can we map it to some matching field in ASIM schema?
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-dns

Reply:
No unfortunately that is something entirely different. It is always filled with the string `Microsoft-Windows-DNSServer`, which doesn't hold any value in this context.